app/services/attendance_service/commands/check_lockout.rb

```diff
@@ -20,6 +20,24 @@ def checkable?
         auth && attendance_root && pseudonym && user && partner_name
       end
 
+      def locked_out?
+        return false if ENV.fetch('ATTENDANCE_LOCKOUT_DISABLED', false)
+        response = HTTParty.get(full_url, headers: { "CanvasAuth" => auth })
+
+        case response.code
+        when 200..299
+          return response['isLockedOut']
```

Comment:
Do we want to return null of this does not exist, or to raise? I think this is fine if we want to return null, but might be better to explode.

Reply:
I've been debating this.

I think the best answer might be explode to sentry but not to the user. Thoughts?